conduct/code_of_conduct.md

```diff
@@ -102,6 +103,30 @@ free to contact the Code of Conduct Committee at
    automatically imply an admission of guilt.
 
 
+## Responding to inappropriate behavior
+
+In some cases, individuals may violate the CoC in online or in-person situations.
+If any community member deems the action to be harmful to the community,
+they may act immediately to defuse the situation.
+This includes actions directed at the situation, rather than at an individual, such
+as locking a thread, *temporarily* suspending a user's account in the
+community forum, calling a meeting break, or generally acting to
+de-escalate the situation. Follow-up your actions by
+[reporting the incident to the CoC committee](#Reporting).
```

Comment:
@rgbkrk what do you think about this wording? better?

Reply:
Much better, thank you.